File: src/main/java/cn/gov/forestry/common/domain/bo/GeneralLogLevelEnum.java
package cn.gov.forestry.common.domain.bo;

import lombok.Getter;

public enum GeneralLogLevelEnum {
    DEBUG("debug", 100),
    INFO("info", 200),
    ERROR("error", 500)
    ;

    @Getter
    private String code;
    @Getter
    private Integer level;

    GeneralLogLevelEnum(String code, Integer level) {
        this.code = code;
        this.level = level;
    }
}

File: src/main/python/cn/gov/forestry/common/domain/bo/general_log_level_enum.py
from enum import Enum


class GeneralLogLevelEnum(Enum):
    DEBUG = ("debug", 100)
    INFO = ("info", 200)
    ERROR = ("error", 500)

    def __init__(self, code: str, level: int):
        self.code = code
        self.level = level


__all__ = ["GeneralLogLevelEnum"]
